notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -152,6 +152,12 @@
                   :selected-translation="selectedTranslation"
                   @change-translation="changeTranslation" />
               </span>
+              <extension-registry-components
+                name="NotesAttachmentButton"
```

Comment:
for more generic name

```suggestion
                name="NotesDetailTitle"
```